src/router/routes/ssoUsers.js

```diff
@@ -0,0 +1,129 @@
+//
+// Reggie
+//
+// Copyright © 2018 Province of British Columbia
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Created by Shelly Xue Han on 2019-01-08.
+//
+
+/* eslint-env es6 */
+
+'use strict';
+
+import { asyncMiddleware, errorWithCode, logger } from '@bcgov/nodejs-common-utils';
+import { Router } from 'express';
+import config from '../../config';
+import { SSO_SUB_URI } from '../../constants';
+import { getSAToken, getUserInfoByEmail, getUserInfoById, updateUser } from '../../libs/sso-utils';
+
+const router = new Router();
+
+// Either get SSO user by email or by ID:
+router.get(
+  '/user',
+  asyncMiddleware(async (req, res) => {
```

Comment:
are there any differences between the two `asyncMiddleware` they look pretty similar.
I believe you can register the middleware by using `router.use(asyncMiddleware)`

Reply:
oops never mind